source/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/UpdateActorHandler.cs

```diff
@@ -54,6 +56,7 @@ public sealed class UpdateActorHandler : IRequestHandler<UpdateActorCommand>
             _overlappingBusinessRolesRuleService = overlappingBusinessRolesRuleService;
             _allowedGridAreasRuleService = allowedGridAreasRuleService;
             _externalActorIdConfigurationService = externalActorIdConfigurationService;
+            _uniqeMarketRoleGridAreaService = uniqeMarketRoleGridAreaService;
```

Comment:
Fix spelling mistake

```suggestion
            _uniqueMarketRoleGridAreaService = uniqueMarketRoleGridAreaService;
```